pkg/roachprod/prometheus/prometheus.go

```diff
@@ -253,7 +253,7 @@ func Init(
 		// NB: when upgrading here, make sure to target a version that picks up this PR:
 		// https://github.com/prometheus/node_exporter/pull/2311
 		// At time of writing, there hasn't been a release in over half a year.
-		if err := c.RepeatRun(ctx, l, l.Stdout, l.Stderr, cfg.NodeExporter,
+		if err := c.Run(ctx, l, l.Stdout, l.Stderr, install.OnNodes(cfg.NodeExporter).WithRetryFn(install.AlwaysRetry),
```

Comment:
Does it literally mean it never fails to retry? Previously, it would max out after 10 attempts albeit with an exponential backoff.